packages/perseus-editor/src/components/graph-points-count-selector.tsx

```diff
@@ -1,9 +1,19 @@
+import {OptionItem, SingleSelect} from "@khanacademy/wonder-blocks-dropdown";
+import {StyleSheet} from "aphrodite";
 import * as React from "react";
 
 import {UNLIMITED, parsePointCount} from "../util/points";
 
 import type {PointValue} from "../util/points";
 
+const NUMERIC_OPTIONS = [...Array(7).keys()].map((n) => (
```

Comment:
🤷‍♂️  This feels off to generate the React components and assign them to a const here.  I haven't seen this done before so I'd defer to you if it's a pattern you've used before.